Client/src/components/footer.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
```

Comment:
If you don't think that you'll add functionality to the footer, you may just want to put the element back into the parent component.  This makes it easier on file management since you know this will be static.